From c0c8805265ce2b40574132558cacf32f9870478d Mon Sep 17 00:00:00 2001
From: Yannick Lanz <yannick.lanz@wifx.net>
Date: Wed, 13 Mar 2024 11:57:41 +0100
Subject: [PATCH] i2c-temp-sensor: back port default i2c device (i2c-1) for
 retro-compatibility on chirpstack-concentratord not supported
 lgw_i2c_set_path.

Signed-off-by: Yannick Lanz <yannick.lanz@wifx.net>
---
 libloragw/src/loragw_hal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libloragw/src/loragw_hal.c b/libloragw/src/loragw_hal.c
index 2587208..dc674e3 100644
--- a/libloragw/src/loragw_hal.c
+++ b/libloragw/src/loragw_hal.c
@@ -210,7 +210,7 @@ static uint8_t ts_addr = 0xFF;
 static int     ad_fd = -1;
 
 /* I2C device */
-static char i2c_device[50];
+static char i2c_device[50] = "/dev/i2c-1";
 
 /* -------------------------------------------------------------------------- */
 /* --- PRIVATE FUNCTIONS DECLARATION ---------------------------------------- */
-- 
2.34.1

